src/openrct2/actions/ClimateSetAction.hpp

```diff
@@ -35,12 +33,12 @@ DEFINE_GAME_ACTION(ClimateSetAction, GAME_COMMAND_SET_CLIMATE, GameActionResult)
     {
         GameAction::Serialise(stream);
 
-        stream << DS_TAG(_climate);
+        stream << DS_TAG(reinterpret_cast<uint8_t&>(_climate));
```

Comment:
I'd prefer we didn't do this. If you see all of the other game actions. I have purposely kept the type as a `uint8_t` and _then validated the value before using_ (okay maybe I didn't always validate it but that was the plan). So `_climate` would  be a `uint8_t` but `climate` in the constructor would be the specific enum class type.